src/components/CollectionsAndPalettes/CollectionsAndPalettes.module.scss

```diff
@@ -1,4 +1,5 @@
 @import "styles/variables.scss";
+@import "styles/mixins.scss";
 .pageWrapper {
```

Comment:
We should add an overflow:hidden for the mobile version for now so that we don't get horizontal scrolling due to the header.